configuration-reference/schema.md

```diff
@@ -1,20 +1,22 @@
-# Schema
+#   Schema
 
 Schema is used to define the names, data types, and other information for the columns of a Pinot table.
 
 The Pinot schema is composed of:
 
-| Field                  | Description                                                                                                                                                                 |
-| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **schemaName**         | Defines the name of the schema. This is usually the same as the table name. The offline and the real-time table of a hybrid table should use the same schema.               |
-| **dimensionFieldSpec** | A dimensionFieldSpec is defined for each dimension column. For more details, scroll down to [DimensionFieldSpec](schema.md#dimensionfieldspec).                             |
-| **metricFieldSpec**    | A metricFieldSpec is defined for each metric column. For more details, scroll down to [MetricFieldSpec](schema.md#metricfieldspec).                                         |
-| **dateTimeFieldSpec**  | A dateTimeFieldSpec is defined for the time columns. There can be multiple time columns. For more details, scroll down to [DateTimeFieldSpec](schema.md#datetimefieldspec). |
+| Field                              | Since | Default   | Description                                                                                                                                                                                                                                           |
+|------------------------------------|-------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **schemaName**                     | -     | mandatory | Defines the name of the schema. This is usually the same as the table name. The offline and the real-time table of a hybrid table should use the same schema.                                                                                         |
```

Comment:
Schema name must be the raw table name. We enforced that in 1.1.0

Reply:
Changed